lib/mpl_toolkits/tests/test_axes_grid.py

```diff
@@ -1,15 +1,26 @@
+from contextlib import ExitStack
 
-from matplotlib.testing.decorators import image_comparison
-from mpl_toolkits.axes_grid1 import ImageGrid
 import numpy as np
+import pytest
+
+import matplotlib as mpl
+from matplotlib.testing.decorators import image_comparison
 import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import ImageGrid
 
 
+# The original version of this test relied on mpl_toolkits's slightly different
+# colorbar implementation; moving to matplotlib's own colorbar implementation
+# caused the small image comparison error.
+@pytest.mark.parametrize("legacy_colorbar", [False, True])
```

Comment:
You don't want to just change the image? 

Otherwise, I'm onboard for this!

Reply:
You know I really, really don't like adding/changing baseline images... hence the check_images_equal decorator, etc.

Reply:
This parametrization makes the test flaky in parallel, as they can overwrite the test file. We only use duplicate file names in the `image_comparison` decorator for that reason.